projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -43,3 +38,30 @@ describe.skip('Profile-tag component', () => {
     });
   });
 });
+describe.skip('login notification', () => {
+  const loginAlias = 'loginNotification';
+  beforeEach(() => {
+    cy.server();
+    cdsHelper.setUpMocks();
+    cy.route('POST', '**/users/current/loginnotification**').as(loginAlias);
+    navigation.visitHomePage({});
+    profileTagHelper.waitForCMSComponents();
+    profileTagHelper.triggerLoaded();
+    profileTagHelper.triggerConsentReferenceLoaded();
+    loginHelper.registerUser();
```

Comment:
Sorry for this since our e2e still has redundancy, and some are not being used, but it would be best to use registerUser from `checkout-flow.ts` for this line.

BUT, I believe you don't need to register a user since you just want to test for failed login and successfully login to verify the endpoint, therefore I suggest to use  the username and password from `checkout-as-persistent-user.ts` as this user will always be in our system (unless someone forgets to import our e2e test data when initializing the backend).

Reply:
Thanks for the hint @bgambocjaviniar! It looks like the login method in the `checkout-as-persistent-user.ts` doesn't navigate to the login page for me. This means I would have to do:
`cy.get(loginHelper.loginLinkSelector).click`
and then
`checkoutAsPersistentUserHelper.loginSuccessfully()`
To avoid having these two separate dependencies, and to have it in one function, I have added a function to the loginHelper. I think this also helps with find-ability, as someone logging in would likely look at the loginHelper first. Let me know if this is ok, or if I should set it to have the 2 dependencies as mentioned above.
Also, it looks like this default user is not on dev-14. At least I am seeing `Bad Credentials. Please log in again`

Reply:
seems like it went down again :x, will restart it and put it in